hack/build-test-bin.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -e
+
+if ! which go; then
+  echo "No go command available"
+  exit 1
+fi
+
+GOPATH="${GOPATH:-~/go}"
+export PATH=$PATH:$GOPATH/bin
+
+if ! which gingko; then
+	echo "Downloading ginkgo tool"
+	go install github.com/onsi/ginkgo/ginkgo
+fi
+
+ginkgo build -mod=vendor ./functests
```

Comment:
nit: add a new line please.

Reply:
another comment can we please move the output file to a bin directory and add it to the `.gitignore` file

Reply:
done